manifests/p/Postman/Postman/9.31/Postman.Postman.locale.en-US.yaml

```
@@ -0,0 +1,27 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Postman.Postman
+PackageVersion: 9.31
```

```suggestion
PackageVersion: 9.31.0
```
This should match Control Panel.